Validation/RecoTrack/test/BuildFile.xml

```diff
@@ -1,4 +1 @@
-<bin name="testMakeTrackValidationPlots" file="TestDriver.cpp">
-  <flags TEST_RUNNER_ARGS="/bin/bash Validation/RecoTrack/test test_makeTrackValidationPlots.sh"/>
-  <use name="FWCore/Utilities"/>
-</bin>
+<test name="testMakeTrackValidationPlots" command="test_makeTrackValidationPlots.sh"/>
```

Comment:
why is this better?

Reply:
You do not need an extra `TestDriver.cpp` file which also means that scram will not compile/build an extra executable.

Reply:
I should have deleted `TestDriver.cpp` too